### PR TITLE
FLUID-6454: Removed title attribute from navbar

### PR DESCRIPTION
**Pull request by sachin10101998**
There's a title attribute on the "Previous" link and the "Next" link. The issue here is that the link has accessible text that will result in screen readers announcing previous twice (e.g. "link, Previous, Previous page" in VoiceOver. It's not recommended that the title attribute be used on links because of inconsistent browser support and the potential for duplicate content being announced by screen readers. 
Reference from MDN: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title#Accessibility_concerns

### CODE REVIEW

**Comment by gtirloni** [comment]
ok to test

**Comment by BlueSlug** [comment]
Thanks for filing this, @sachin10101998. I've left a comment on the [Jira issue](https://issues.fluidproject.org/browse/FLUID-6454?focusedCommentId=41433&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-41433) detailing some limitations with this particular fix. I recommend using `aria-label` instead, but it would be good to wait for confirmation from @jobara or @greatislander.

File: demos/pager/css/pager.css [hunk 1]
```diff
@@ -127,3 +127,14 @@
...
+ /***********************************************
+ * Styles for pager previous and next page symbol
+ */
+ .demo-pager-previous-character::before {
+     content: "<";
+ }
+
+ .demo-pager-next-character::after {
+     content: ">";
+ }
```
Comment (jobara) [inline demos/pager/css/pager.css:140]: please add an empty new line character at the end of the file.

Comment (sachin10101998) [reply demos/pager/css/pager.css]: Done

Comment (sachin10101998) [reply demos/pager/css/pager.css]: Done

**Comment by jobara** [comment]
@sachin10101998 after testing with VoiceOver on macOS I've found that my original suggestion didn't work. I've left an updated [comment](https://issues.fluidproject.org/browse/FLUID-6454?focusedCommentId=41534&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-41534) on the issue. Also, when you make the update, could you please also updated the [Pager Examples](https://github.com/fluid-project/infusion/tree/master/examples/components/pager)?

File: demos/pager/index.html [hunk 2]
```diff
@@ -62,15 +62,15 @@ <h1>School Roster</h1>
 
             <div class="flc-pager-top">
                 <ul class="demo-pager-bar fl-force-left fl-focus">
-                    <li class="flc-pager-previous demo-pager-previous"><a href="#" title="Previous Page">&lt; Previous</a></li>
+                    <li class="flc-pager-previous demo-pager-previous"><span class="demo-pager-previous-character" aria-hidden="true">&lt;</span><a href="#"> Previous</a></li>
```
Comment (jobara) [inline demos/pager/index.html:65]: I don't think you need the class names anymore with the characters added in directly.

Comment (jobara) [reply demos/pager/index.html:65]: the `<` and `>` symbols should be contained within the `<a>`. If that isn't workable with the `aria-hidden="true"` we may need to find an alternative solution.

Comment (sachin10101998) [reply demos/pager/index.html:65]: Yes @jobara , with aria-hidden true, we can't put things that are focusable inside the span element. We'll have to go with icons as discussed after making changes in the pager component.

**Comment by amb26** [comment]
Closing since the pager component will be removed from the next Infusion release

**Pull request closed without merging**